Sources/Core/iOS/NetworkObserver.swift

```diff
@@ -82,19 +82,19 @@ private class NetworkIndicatorController {
     // Public API
 
     func networkActivityDidStart() {
-        assert(Thread.isMainThread(), "Altering network activity indicator state can only be done on the main thread.")
+        assert(Thread.isMainThread, "Altering network activity indicator state can only be done on the main thread.")
```

Comment:
Yep -  I will revert the current `swift/3.0` branch and re-create from `develop` and @jshier can PR to it.